lib/storekit/client.rb

```diff
@@ -28,12 +28,13 @@ def verify!(base64_payload)
       resp = @http.request(req)
 
       decoded = JSON.parse resp.body
+      @last_response = ReceiptResponse.new(decoded)
```

Comment:
This should behave as a pure function, so I disagree with storing the last response.